Spec/RestClient.swift

```diff
@@ -839,6 +839,7 @@ class RestClient: QuickSpec {
                         }
                         
                         expect(testHTTPExecutor.requests).to(haveCount(Int(1 + options.httpMaxRetryCount)))
+                        expect((testHTTPExecutor.requests.count) < (_fallbackHosts.count + 1)).to(beTrue())
```

Comment:
Why not `expect(testHTTPExecutor.requests).to(haveCount(Int(1 + options.httpMaxRetryCount)))` though?